src/Faker/Provider/me_ME/Address.php

```diff
@@ -111,17 +111,23 @@ public function cityName()
 
     /**
      * @example '77.147489'
+     * @param float|int $min
+     * @param float|int $max
+     * @return float Uses signed degrees format (returns a float number between -90 and 90)
      */
-    public static function latitude()
+    public static function latitude($min = 42.43, $max = 42.45)
```

Comment:
`latitude()` and `longitude()` shouldn't be localized in the first place. This is a mistake in the existing `me_ME` formatters. Please create a new formatter, like `localGeolocation()`, for that purpose. That way, you can generate latite and longitude within a polygon.